app/scripts/controllers/common.js

```diff
@@ -153,8 +153,10 @@ const commons = angular.module('common', ['stats', 'udc', 'events', 'sql'])
     updateClusterInfo();
 
     // bind tooltips
-    $('[rel=tooltip]').tooltip({
-      placement: 'bottom'
+    $(document).ready(function() {
```

Comment:
Why is waiting for the document ready needed now? Or was it always causing issues?

Reply:
i forgot to remove this, I will push a fix up.
the waiting for document ready is not needed when jquery and bootstrap are loaded properly

Reply:
Ah ok! Cool